.travis.yml

```diff
@@ -77,14 +83,19 @@ matrix:
     include:
         - otp_release: 20.0
           env: PRESET=dialyzer_only
+               SKIP_RELEASE=1 SKIP_BUILD_TESTS=1 SKIP_COV=1 SKIP_REPORT_UPLOAD=1
         - otp_release: 20.0
           env: PRESET=pgsql_mnesia DB=pgsql REL_CONFIG="with-pgsql with-jingle-sip"
         - otp_release: 18.3
           env: PRESET=riak_mnesia DB=riak REL_CONFIG=with-riak
-        - otp_release: 19.3
+        - language: generic
```

Comment:
How much time does this change save?

Reply:
10 seconds * two builds. But I just don't want to install stuff we don't need for packaging.